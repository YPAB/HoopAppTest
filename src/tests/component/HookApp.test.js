import React from 'react';

import '@testing-library/jest-dom';
import {shallow} from 'enzyme';
import { HookApp } from '../../HookApp';



describe('Test Componente Hook App',() => {

    test('Debe mostrar <HookApp></HookApp> correctamente', () => {

        const wrapper = shallow(<HookApp/>)


        expect(wrapper).toMatchSnapshot();

    });




})