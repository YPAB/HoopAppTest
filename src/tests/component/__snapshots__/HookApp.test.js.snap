// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Componente Hook App Debe mostrar <HookApp></HookApp> correctamente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
